README.md

```diff
@@ -660,232 +847,374 @@ commit and push your work.
+
+```java
+<<<<<<< chris-greeting
+    builder.append(chrisSaysHello());
+=======
+    builder.append(patSaysHello());
+>>>>>>> main
+```
+
+What this rather odd syntax says is that Chris's `chris-greeting` branch has
+a line that calls `chrisSaysHello()`, where `main` (which has Pat's changes)
+has a line that calls `patSaysHello()`. Everything between the `<<<<` line
+and the `====` line is what things look like in the `chris-greeting` branch,
+and everything between the `====` line and the `>>>>` line are what things
+look like in the `main` branch.
+
+There are numerous ways this conflict could have come about, such as:
+
+* Chris wants to change the line from calling `patSaysHello()` to
+  instead called the (supposedly different) method `chrisSaysHello()`.
+* Chris has renamed `patSaysHello()` to `chrisSaysHello()`, and wants to change
+  the call to match this renaming.
+* Chris wants to make _both_ function calls (in some order, which `git` couldn't
+  begin to guess).
+
+Here `git can't tell which of these happened, and thus can't "know" how to
+handle the merge. Thus it punts it to us, the human users to sort out. Sometimes
+it's obvious to us what to do, but sometimes it's really _not_ clear what the
+right course of action is. :warning: DO NOT JUST THROW AWAY OTHER PEOPLE'S
+WORK TO RESOLVE A MERGE CONFLICT. This is a place where you really want to
+take a moment and contact other people on your team to decide what is the best course
+of action.
+
+In this case the last option is the case, and we better call `chrisSaysHello()`
+first so that at least those two calls are in alphabetical order.
+
+Chris would use the web editor to make those changes, deleting the `<<<<`, `====`,
+and `>>>>` lines, leaving the two calls in the desired order. When things look good,
+Chris clicks "Mark as resolved", and then the button to commit the merged changes.
+
+:warning: This is a _very_ simple example of a merge conflict. They can be much more
+complex than this, and trickier to resolve. The Pro version of GitKraken (which you
+got through the GitHub Student Pack) actually has a very nice GUI for resolving merge
+conflicts, and you'd probably rather use that for more complex conflicts.
+
+:warning: :bangbang: At this point Chris (or you if you've been doing something
+similar) will need to pull these changes back down to your copy if you want
+or need to make changes to your branch. If, for example, Chris realizes here
+that the tests are failing because they have the alphabetical order wrong,
+they'll need to pull down these changes, fix the alphabetical order, push back
+up, and attempt to merge again.
+
+##### You need a review
+
+Now you've finally gotten your code up-to-date and all the checks pass. You
+still can't merge because of a big red X and the message "Review required".
+This is because we're following a common "best practice" and requiring at
+least one code review _by another person_ for each pull request.
```

Comment:
Are we changing this to two code reviews? If so:
```suggestion
least two code review _by other people_ for each pull request.
```

Reply:
Good catch – thanks!